src/Core/Services/SyncService.cs

```diff
@@ -374,7 +374,11 @@ private async Task SyncPoliciesAsync(List<PolicyResponse> response)
             await _policyService.Replace(policies);
         }
 
-        private Task SyncSendsAsync(string userId, List<SendResponse> sends) =>
-            _sendService.ReplaceAsync(sends.ToDictionary(s => userId, s => new SendData(s, userId)));
+        private async Task SyncSendsAsync(string userId, List<SendResponse> response)
+        {
+            var sends = response?.ToDictionary(s => s.Id, s => new SendData(s, userId)) ?? 
+                new Dictionary<string, SendData>();
```

Comment:
default initializers for collections that will always be empty should be zero `new Dictionary<TKey, TValue>(0)` or `new List<T>(0)`, similar to using `Array<T>.Empty()` which keeps memory allocations off of the large object heap and less for the garbage collector to do.

Reply:
Makes sense, but from what I can tell it looks like the default capacity is already `0` when an `Int32` isn't provided.  Is that correct?